src/app.js

```
@@ -1 +1,54 @@
+/* eslint-disable no-console */
 // write code here
+
+const fs = require('fs');
+const path = require('path');
+
+const moveFiles = () => {
+  const params = process.argv.slice(2);
+
+  if (params.length < 2) {
+    console.error('There are missing arguments');
+
+    return;
+  }
+
+  const source = params[0];
+  let destination = params[1];
+
+  if (!fs.existsSync(source)) {
+    console.error('Source not exist');
+
+    return;
+  }
+
+  const isDestDir =
+    destination.endsWith('/') ||
+    (fs.existsSync(destination) && fs.statSync(destination).isDirectory());
+
+  if (isDestDir) {
+    const filename = path.basename(source);
+
+    destination = path.join(destination, filename);
+  }
+
+  const targetDir = path.dirname(destination);
+
+  if (!fs.existsSync(targetDir)) {
+    console.error('Destination directory does not exist');
+
+    return;
+  }
+
+  if (source === destination) {
+    return;
```

The code checks if source and destination are the same, but it may be better to throw an error instead of silently returning. Consider to log an error to the console.